src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.incidents.models import IncidentActivity, IncidentActivityType, IncidentStatus
+from sentry.models import Project
+
+
+class TeamAlertsTriggeredEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a time-bucketed (by day) count of triggered alerts owned by a given team.
+        """
+        project_list = Project.objects.get_for_team_ids([team.id])
+        owner_ids = [team.actor_id] + [om.user.actor_id for om in team.member_set]
+        start, end = get_date_range_from_params(request.GET)
+        bucketed_alert_counts = list(
+            IncidentActivity.objects.filter(
+                type__in=[
+                    IncidentActivityType.CREATED.value,
+                    IncidentActivityType.STATUS_CHANGE.value,
+                ],
+                value__in=[
+                    None,
```

Comment:
Might be more correct to have:

```
.filter(
   Q(IncidentActivityType.CREATED.value) | Q(IncidentActivityType.STATUS_CHANGE.value, value__in=<statuses>)
)
```

Not sure if that'll end up making it slow, but I think this table is relatively small so it should be ok.

Reply:
```
--WARNING! ERRORS ENCOUNTERED DURING SQL PARSING!
SELECT "sentry_incidentactivity"."id"
	,"sentry_incidentactivity"."incident_id"
	,"sentry_incidentactivity"."user_id"
	,"sentry_incidentactivity"."type"
	,"sentry_incidentactivity"."value"
	,"sentry_incidentactivity"."previous_value"
	,"sentry_incidentactivity"."comment"
	,"sentry_incidentactivity"."date_added"
FROM "sentry_incidentactivity"
INNER JOIN "sentry_incident" ON ("sentry_incidentactivity"."incident_id" = "sentry_incident"."id")
INNER JOIN "sentry_alertrule" ON ("sentry_incident"."alert_rule_id" = "sentry_alertrule"."id")
INNER JOIN "sentry_incidentproject" ON ("sentry_incident"."id" = "sentry_incidentproject"."incident_id")
WHERE (
		(
			"sentry_incidentactivity"."type" = 1
			OR (
				"sentry_incidentactivity"."type" = 2
				AND "sentry_incidentactivity"."value" IN (
					None
					,IncidentStatus.OPEN
					,IncidentStatus.CRITICAL
					,IncidentStatus.WARNING
					)
				)
			)
		AND "sentry_incidentactivity"."date_added" >= 2021 - 09 - 14 19 : 39 : 33.437298 + 00 : 00
		AND "sentry_incidentactivity"."date_added" <= 2021 - 09 - 28 19 : 39 : 33.437298 + 00 : 00
		AND "sentry_alertrule"."owner_id" IN (
			6
			,5
			)
		AND "sentry_incidentproject"."project_id" IN (
			SELECT U0."id"
			FROM "sentry_project" U0
			INNER JOIN "sentry_projectteam" U1 ON (U0."id" = U1."project_id")
			WHERE (
					U0."status" = 0
					AND U1."team_id" IN (4)
					)
			)
		)
```
Feels better